src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,30 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+import helpers from '../helpers';
+import services from '../services';
+
+config();
+
+const { SECRET_KEY } = process.env;
+
+const { responseMessage } = helpers;
+const { findUser } = services;
+
+export default (request, response, next) => {
+  const token = request.headers.authorization || request.query.token;
+
+  if (token) {
```

Comment:
instead of having nested if, can you do `if (!token) { return responseMessage(response, 401, { error: 'no token provided' }); }` then do your jwt verification

Reply:
Noted, would work on it